README.md

```diff
@@ -124,15 +70,20 @@ jobs:
         dockerfiles: |
           ./Dockerfile
 
-    - name: Push To Quay
+    - name: Log in to quay.io
```

Comment:
Let's leave podman-login out of the example, but put a note in a comment that points to it.